### ISSUE TITLE
a11y improvements

### ISSUE THREAD

**Issue by i-like-robots** [Financial-Times/next-article#549]
- [x] Article content requires `role="main"` and ID E.G. `id="site-content"`
- [x] Story title should be H1 element
- [x] Subtitle [should not be a heading element](http://www.w3.org/TR/html5/common-idioms.html#sub-head)
- [x] Main category link needs `aria-label` with appropriate description
- [x] Topics mentioned link list should be an aside w/ `role="complementary"`
- [ ] Main article image has no alt text. Use `aria-labelledby="*figcaption element ID*"`
- [x] Promoboxes to other articles should be an aside w/ `role="complementary"` and `aria-label="Related content"`
- [x] More on/Related topic areas should be each have `role="region'`
- [x] `<abbr>` elements require text content ([here](https://github.com/Financial-Times/next-article/blob/82e170fa76bf7edb90dbf447ca46dcb2b4aa145c/views/partials/reading-list.html#L3))

**Comment by adamsilver** [comment]
Moved "Main article image has no alt text. Use aria-labelledby="*figcaption element ID*"" into separate issue.  Rest are done.
